resources/after.sh

```diff
@@ -3,3 +3,167 @@
 # If you would like to do some extra provisioning you may
 # add any commands you wish to this file and they will
 # be run after the Homestead machine is provisioned.
+
+#
+## Global setup
+#
+# Avoid apt-get calls to be interactive
+export DEBIAN_FRONTEND=noninteractive
+
+# Avoid mysql commands to show a warning for using an inline password
+echo "secret" > mysql_config_editor set --login-path=local --host=localhost --user=homestead --password
+
+# Set global output colors
+NC="\033[0m"
+WHITE="\033[0;0m"
+GREEN="\033[0;37m"
+WHITE_ON_PURPLE="\033[37;45m"
+WHITE_ON_GREEN="\033[37;42m"
+
+# Constants
+ES_PORT=9200
+
+#
+## Functions
+#
+displayTitle () {
+    echo " "
+    echo " "
+    echo " "
+    echo "${WHITE_ON_PURPLE}  $1  ${NC}"
+    echo " "
+}
+
+displayOkMessage () {
+    echo " "
+    echo "${WHITE_ON_GREEN}  OK  ${NC}"
+}
+
+displayMessage () {
+    echo "${WHITE}$1${NC}"
+}
+
+displaySpacedMessage () {
+    echo " "
+    displayMessage "$1"
+}
+
+displayServerInfo () {
+    echo " "
+    echo " "
+    echo " "
+    displayMessage "--------------------------------------"
+    echo "${WHITE_ON_GREEN}  ALL READY!  ${NC}"
+    echo " "
+    displayMessage "bower version: `bower -v`"
+    displayMessage "composer version: `composer -V | grep -Po '\d+(\.\d+)+'`"
+    displayMessage "elasticsearch version: `curl -s -XGET http://localhost:$ES_PORT | grep -E \"\\\"number\\\"\" | grep -Po '\d+(\.\d+)+'`"
+    displayMessage "php version: `php -v | grep -Po 'PHP \d+(\.\d+)+' | sed -e 's/PHP //'`"
+    displayMessage "mysql version: `mysql -Vmysql -V | grep -Po 'Distrib \d+(\.\d+)+' | sed -e 's/Distrib //g'`"
+    displayMessage "node version: `node -v`"
```

Comment:
I'm a bit wary of installing and using Node from homestead as there's other Node projects which don't depend on Homestead - such as the OpenChargeNative app etc. - think we should omit these bits from the set up.